### ISSUE TITLE
Screenreader accessibility improvements

### ISSUE THREAD

**Issue by VIPPotato** [Valkirie/HandheldCompanion#1379]
There are some areas in which it would be nice to have screenreader improvements so blind and visually impaired people can use the app more easily on their handhelds: 
In order to test screenreader accessibility, you either can turn on narrator by pressing win+ctrl+enter, or download a free screenreader called nvda from nvaccess.com. 
After running it, open handheld companion and try navigating with tab and shift+tab and listen carefully what it says. 
Here's a list of things which should be fixed: 
- Each item like toggle buttons, combo boxes, sliders etc should have accessibility labels attached, that way screenreader users know instantly what they are changing with out the need to use an object navigation in their screenreader of choice. 
- By navigating with tab, shift+tab and arrows with a screenreader turned on, you'll notice names of most items aren't being spoken. For example, in the library tab a combo box responsible for sorting is being announced as "combo box  Title  collapsed" instead of "sort by combo box  Title  collapsed". 
Settings tab and one tab to the left and one to the right are being announced as "NavigationViewItem  tab" instead of its name. 
- When one of the main tabs is selected (controller, library, device, performance, profiles, overlay or hotkeys" it is impossible to focus on any content inside that tab by pressing the tab key, the only way is to cycle back with shift+tab which is a huge inconvenience. 
For example, let's say we want to disable CPU boost for some reason. We tab until finding tabs, and search for performance with left and right arrows. Then after focusing on it, normally we would use tab to find what we need, but the screenreader focuses on a set of 3 unlabelled tabs, then we find an unlabelled list of notifications, and pressing tab once again goes back to main tabs where we selected performance. But now if we press shift tab we notice there actually are options like fan mode, CPU parking mode etc. I'm not sure what causes this issue, but I've noticed it in some other apps made with xaml, but that's not how it should behave.
